apps/src/applab/applab.js

```diff
@@ -367,12 +367,24 @@ function shouldRenderFooter() {
   return studioApp.share;
 }
 
+const APPLAB_URL_PATTERN = /^(.*\/projects\/applab\/[A-Za-z0-9-]+)\/.*/;
```

Comment:
Do you also want this to match a URL that's already missing the trailing slash, like `https://studio.code.org/projects/applab/<key>`?  Looks like the trailing slash is required right now.